crates/bevy_sprite/src/lib.rs

```diff
@@ -88,7 +88,7 @@ impl Plugin for SpritePlugin {
                     // In practice, they run independently since `bevy_render::camera_update_system`
                     // will only ever observe its own render target, and `update_text2d_layout`
                     // will never modify a pre-existing `Image` asset.
```

Comment:
The comment about the ambiguity needs updating.

Reply:
There's no more ambiguity I guess, so just deleted it, its meaning didn't seem very clear.